manifests/06-deployment.yaml

```diff
@@ -18,6 +19,7 @@ spec:
     metadata:
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+        capability.openshift.io/name: insights
```

Comment:
Is that required here on the template level please (same above for ibm cloud deployment manifest)? I am looking at https://github.com/openshift/cluster-samples-operator/blob/master/manifests/06-operator.yaml and it doesn't seem so.

Reply:
Well, now this is a good question...
https://github.com/honza/cluster-baremetal-operator/blob/8f43aca6a89b151af287fa5532749f93c26fc238/manifests/0000_31_cluster-baremetal-operator_06_deployment.yaml#L21

Reply:
But I don't think that it is... TBH

Reply:
I don't know. @wking is the annotation required on the template level as well please?

Reply:
The cluster-version operator is just looking at the top-level `metadata.annotations` in each manifest.  It doesn't drill down into `spec` when trying to decide whether a given YAML resource should be reconciled in the current cluster.  So +1 to dropping this line.

Reply:
Thank you.